core/src/env/traits.rs

```diff
@@ -20,16 +20,30 @@ use crate::{
 };
 use parity_codec::Codec;
 
-/// The environmental types usable by contracts defined with pDSL.
+#[cfg(not(feature = "test-env"))]
+/// The environmental types usable by contracts defined with ink!.
 pub trait EnvTypes {
     /// The type of an address.
-    type AccountId: Codec + PartialEq + Eq;
+    type AccountId: Codec + Clone + PartialEq + Eq;
```

Comment:
Could you also add a conditional trait bound to `Debug` if `std` is enabled?
I think this could be great for testability using `println` as we sometimes have to.
Or am I misunderstanding that we don't need to add this trait bound for practical uses since the concrete types are already known upon usage?

Reply:
Yeah we don't need the trait bound at this level, so long as the concrete type is implementing `Debug` it will work.

Note that below is a definition for the `test-env` feature with a `Debug` bound, which is required for test asserts where we don't have the concrete type.